### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ruchikaa123**
add code for calculating the first 10 odd numbers using C++ language in comparison of language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ map ((+1) . (*2)) [0..9] -- alternative thanks to Alexandru Dinu
 ```ijs
 1+2*i.10
 ```
+**C++**:
+```c++
+for(int i=0;i<10;i++){
+cout<<(2 * i )+ 1<<"\n";}
```
Comment (codereport) [inline README.md:67]: If you want to add C++, can we add a C++20 ranges version? With a compiler explorer link?

**Review by codereport — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sebanisu) [reply README.md:67]: like this? 
```c++
#include <ranges>
#include <algorithm>
#include <iostream>
int main()
{
    std::ranges::for_each(std::ranges::iota_view(0,10),[](auto i){std::cout << (2 * i)+1 << '\n';});
}
```
https://godbolt.org/z/orMvrn

Comment (codereport) [reply README.md:67]: Close, `views::iota` and `views::transform` are preferable imo.

Comment (codereport) [inline README.md:67]:
```suggestion
// https://godbolt.org/z/5r7aEo
iota(0, 10) | transform([](auto e) { return e * 2 + 1; })
```

**Review by codereport — CHANGES_REQUESTED** [review]

**Review by codereport — APPROVED** [review]

**Pull request merged**